tests/InMemoryStorageService.ts

```diff
@@ -12,7 +16,7 @@ interface StorageRecord {
   id: string
 }
 
-@scoped(Lifecycle.ContainerScoped)
+@injectable()
```

Comment:
Seeing it like this I might be a bit worried about the name, it is quite vague. Would it make sense to you to name it a bit mor explicit like `@injectablePlugin` or something that when you use afj and tsyringe in the same project there are no collisions.

Reply:
This is `@injectable` from tsyringe itself, just re-exported so you don't need to add the dependency. Or do you mean export the tsyringe `@injecatble` under a different name?

Reply:
Ah okay. It was more that it would not collide with your own usage of `tsyringe` in your project where you also use afj. But I don't think that that would be an issue.

Reply:
Ah like that. I did it like this so you wouldn't have to import tsyringe at all in your project (if only using tsyringe for AFJ)